ruby/lib/minitest/queue/runner.rb

```diff
@@ -362,34 +373,34 @@ def parser
             Defaults to 0.
           EOS
           opts.separator ""
-          opts.on('--max-requeues MAX', *help) do |max|
-            queue_config.max_requeues = Integer(max)
+          opts.on('--max-requeues MAX', Integer, help) do |max|
+            queue_config.max_requeues = max
           end
 
           help = <<~EOS
             Defines how long ci-queue should maximally run in seconds
             Defaults to none.
           EOS
           opts.separator ""
-          opts.on('--max-duration SECONDS', *help) do |max|
-            queue_config.max_duration = Integer(max)
+          opts.on('--max-duration SECONDS', Integer, help) do |max|
```

Comment:
this should be in milliseconds and not seconds as the unit is too big.

Reply:
This is old code. If we are changing it, we should change it in a separate PR. This will also be a breaking API change for anybody using the gem.

Reply:
We haven't released the feature yet so I wouldn't be concerned with breaking the API. If you want to tackle it in a separate PR please create an issue and add it to our kanban board. So we can track it :)

Reply:
Oh shooot. Forget what I saw. I commented on the wrong option.